app/femr/data/DataModelMapper.java

```diff
@@ -363,6 +363,7 @@ public IPatient createPatient(int userID, String firstName, String lastName, Dat
         patient.setUserId(userID);
         patient.setFirstName(firstName);
         patient.setLastName(lastName);
+        patient.setPhoneNumber(phoneNumber);
```

Comment:
needs a NULL check otherwise the phone_number column will always be an empty string instead of NULL